lib/rubocop/cop/layout/empty_lines_around_exception_handling_keywords.rb

```diff
@@ -128,7 +128,7 @@ def keyword_locations_in_rescue(node)
         end
 
         def keyword_locations_in_ensure(node)
-          ensure_body, = *node
+          ensure_body, = *node # rubocop:disable InternalAffairs/NodeDestructuring
```

Comment:
`EnsureNode#body` is available, so you should be able to use `node.body` here.

Reply:
`node.body` returns `node.children[1]`, but what’s called `ensure_body` here is actually `node.children[0]`. The wording confuses me.

I might instead change this to
```rb
rescue_body_without_ensure = node.children.first
```

Reply:
Yeah I opened rubocop/rubocop-ast#334 because I don't think `EnsureNode#body` is named consistently.

Reply:
Ah. I’ll wait for #333 and use `rescue_body = node.rescue_node` 😅